frontend/src/pages/game.rs

```diff
@@ -79,8 +79,8 @@ impl Component for Game {
 
     fn create(_ctx: &Context<Self>) -> Self {
         let game: GameStore = gloo_storage::LocalStorage::get("timestampPH").unwrap();
-
-        let ws = WebSocket::open("wss://echo.websocket.org").unwrap();
+        let url = format!("ws://localhost:6000/ws/token/{}",game.token);
```

Comment:
Toto zařaď do routes - pokud nám při demonstraci nebude chtít OS dát port 6000, budeme ho muset změnit -> je fajn mít jedno místo, na kterém to změníš i na frontendu